### DISCUSSION TITLE
Mail作业中间文件

### DISCUSSION THREAD

**Discussion by YiHui-Liu** [physics-data/faq#124] [Q&A] [answered]
自己生成的中间文件，在grade.sh的clean中不会被清除，导致报错，但是在makefile的clean中写了清除中间文件
是否可以更改grade.sh的clean函数为`make clean`

**Comment by heroxbd** [comment]
@ZenithalHourlyRate

**Comment by heroxbd** [comment]
如果 "在grade.sh的clean中不会被清除“ 会导致什么报错？

**Comment by ZenithalHourlyRate** [comment, marked as answer]
可以把自己的中间文件加在 grade.sh 的 `function clean {` 中

> 于是，本题中允许出现题目中未提到的文件，供同学自己使用，只需要 clean 的时候会被清理即可。

https://physics-data.meow.plus/faq/error/mail/ 此处的这句话也有这个意思
